state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -43,6 +43,13 @@ pub use pallet::*;
 use sp_runtime::traits::UniqueSaturatedInto;
 use sp_std::{vec, vec::Vec};
 
+#[derive(Clone, Debug, PartialEq, Eq, Encode, Decode, TypeInfo)]
+pub struct BoostableDeposit<C: Chain> {
```

Comment:
When we have the extrinsic, if someone references this, how can we know what action to take? We need to store the deposit address here so that we can look up `DepositChannelLookup` to find the action. Also probably just useful for product anyway.